test/userController.test.js

```diff
@@ -71,9 +71,28 @@ afterAll(async () => {
   await mongodb.stop();
 });
 
+beforeEach(async () => {
+  // Delete all users with the same email before creating the test user
+  await User.deleteMany({ email: testUserCredentials.email });
+
+  // Create a test user before each test
+  testUser = await User.create({
+    name: 'ava',
+    username: 'ava',
+    email: 'ava@ava.com',
+    password: 'secret',
+    role: 'user',
+  });
+});
+
+afterEach(async () => {
+  // Delete the test user after each test
+  await User.deleteMany({});
+});
```

Comment:
Same comment as in the other usercontroller test PR about simplifying this by having only one step for `deleteMany` and removing the redundant test user creation from the `beforeAll` block

Reply:
<img width="762" alt="jest" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/f1f7d71e-00ee-4e3e-a3d0-4436bc7e2f90">
1 failed, can't fix😒